modules/dmrpp_module/Chunk.h

```diff
@@ -84,8 +84,21 @@ class Chunk {
     unsigned long long d_offset{0};
     unsigned long long direct_io_offset{0};
     unsigned int d_filter_mask{0};
+
+    // The following two members mean this chunk is actually a linked block.
+    // We don't need to remember chunk position but to remember the linked block index.
     bool linked_block{false};
     unsigned int linked_block_index {0};
+
+    // The following two memebers mean this chunk contains multiple linked blocks.
+    // This is still a chunk but inside this chunk it contains multiple linked blocks.
+    // We will record this chunk multiple times with different multi_linked_block_index_in_dmrpp_file.
+    // This is for generating the dmrpp file. 
+    bool multi_linked_blocks{false};
```

Comment:
Let's keep the "{false}" initializer for now, but I wanted to point out that it's a an awkward way to initialize scalar values. We could be using "multi_linked_blocks= false". I think the reason the code from about the time Chunk was first written uses the "{}" syntax is because using "=" with objects means using operator=() which could be specially defined for the class. But for scalar values like '0' and 'false', that's not the case.

Reply:
I change  a bunch of linked block members to the normal syntax.